README.md

```diff
@@ -0,0 +1,37 @@
+# Medicine Project
+
+
+## Requirements
+
+* [Node](https://nodejs.org/download/) (v0.10.x or greater)
+* [npm](https://www.npmjs.com/) package manager
+
+## Installation
+
+```bash
+git clone https://github.com/chloehwang/medicine-viget.git
+cd medicine-viget
+npm install
```

Comment:
have you tried out yarn? (https://yarnpkg.com/en/) It's what we've been using lately and what `blendid` uses by default. No need to change it here, just a heads up about that.